src/extensions/renderer/canvas/drawing-label-text.js

```diff
@@ -164,38 +164,38 @@ CRp.getTextAngle = function( ele, prefix ){
   return theta;
 };
 
-CRp.drawText = function( context, ele, prefix, applyRotation = true, useEleOpacity = true ){
+CRp.drawText = function (context, ele, prefix, applyRotation = true, useEleOpacity = true) {
   let _p = ele._private;
   let rscratch = _p.rscratch;
   let parentOpacity = useEleOpacity ? ele.effectiveOpacity() : 1;
 
-  if( useEleOpacity && (parentOpacity === 0 || ele.pstyle( 'text-opacity' ).value === 0) ){
+  if (useEleOpacity && (parentOpacity === 0 || ele.pstyle('text-opacity').value === 0)) {
     return;
   }
 
   // use 'main' as an alias for the main label (i.e. null prefix)
-  if( prefix === 'main' ){ prefix = null; }
+  if (prefix === 'main') { prefix = null; }
 
-  let textX = util.getPrefixedProperty( rscratch, 'labelX', prefix );
-  let textY = util.getPrefixedProperty( rscratch, 'labelY', prefix );
+  let textX = util.getPrefixedProperty(rscratch, 'labelX', prefix);
+  let textY = util.getPrefixedProperty(rscratch, 'labelY', prefix);
   let orgTextX, orgTextY; // used for rotation
-  let text = this.getLabelText( ele, prefix );
+  let text = this.getLabelText(ele, prefix);
+
+  if (text != null && text !== '' && !isNaN(textX) && !isNaN(textY)) {
 
-  if( text != null && text !== '' && !isNaN( textX ) && !isNaN( textY ) ){
-    this.setupTextStyle( context, ele, useEleOpacity );
```

Comment:
Would you please revert any non-functional, formatting-related changes in your PR?  You're welcome to make formatting changes, but they should be made in a separate PR which includes only formatting changes.

It's not tractable to review your proposed changes with a noisy diff.